src/test/ui/nll/borrowed-universal-error.rs

```diff
@@ -0,0 +1,22 @@
+// Copyright 2018 The Rust Project Developers. See the COPYRIGHT
+// file at the top-level directory of this distribution and at
+// http://rust-lang.org/COPYRIGHT.
+//
+// Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+// http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+// <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your
+// option. This file may not be copied, modified, or distributed
+// except according to those terms.
+
+#![feature(nll)]
+#![allow(warnings)]
+
+fn gimme(x: &(u32,)) -> &u32 { &x.0 }
+
+fn foo<'a>(x: &'a (u32,)) -> &'a u32 {
+    let v = 22;
+    gimme(&(v,))
+}
```

Comment:
Current output for this test:

```
lunch-box. rustc +nightly borrowed-universal-error.rs -Znll-dump-cause
error[E0597]: borrowed value does not live long enough
  --> borrowed-universal-error.rs:18:12
   |
18 |     gimme(&(v,))
   |            ^^^^ temporary value does not live long enough
19 | }
   | - temporary value only lives until here
   |
   = note: borrowed value must be valid for lifetime '_#3r...
   = note: because of an outlives relation created at `bb0[11]`
           because of an outlives relation created at `bb0[12]`
           because of an outlives relation created at `bb0[12]`
           because of an outlives relation created at `bb0[12]`
           because of an outlives relation created at `bb2[0]`
           because of an outlives relation created at `bb2[1]`
           because of an outlives relation created at `bb2[1]`
           because of an outlives relation created at `bb0[0]`
           because `'_#1r` is universally quantified
```

Reply:
this is clearly missing *some* mention of `'a`. Compare to the AST checker:

```
lunch-box. rustc +nightly borrowed-universal-error.rs -Znll-dump-cause
error[E0597]: borrowed value does not live long enough
  --> borrowed-universal-error.rs:17:12
   |
17 |     gimme(&(v,))
   |            ^^^^ temporary value does not live long enough
18 | }
   | - temporary value only lives until here
   |
note: borrowed value must be valid for the lifetime 'a as defined on the function body at 15:1...
  --> borrowed-universal-error.rs:15:1
   |
15 | fn foo<'a>(x: &'a (u32,)) -> &'a u32 {
   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

If we invoke `note_and_explain_region` from `explain_why_borrow_contains_point` in the case of a `LiveUniversal` root cause, then we would get output similar to the AST-checker.